sdk/examples/how_tos/native_tokens/burn.rs

```diff
@@ -37,52 +42,53 @@ async fn main() -> Result<()> {
     // May want to ensure the account is synced before sending a transaction.
     let balance = account.sync(None).await?;
 
-    // Get a token with sufficient balance
-    if let Some(token_id) = balance
-        .native_tokens()
-        .iter()
-        .find(|t| t.available() >= U256::from(MIN_AVAILABLE_AMOUNT))
-        .map(|t| t.token_id())
-    {
-        let available_balance = balance
-            .native_tokens()
-            .iter()
-            .find(|t| t.token_id() == token_id)
-            .unwrap()
-            .available();
-        println!("Balance before burning: {available_balance:?}");
+    // Take the given token id, or use a default.
+    let token_id = std::env::args()
+        .nth(1)
+        .map(|s| s.parse::<TokenId>().expect("invalid token id"))
+        .unwrap_or_else(|| TokenId::from(*balance.foundries().first().unwrap()));
+
+    if let Some(native_token_balance) = balance.native_tokens().iter().find(|native_token| {
+        native_token.token_id() == &token_id && native_token.available() >= U256::from(MIN_AVAILABLE_AMOUNT)
+    }) {
+        println!("Balance before burning:\n{native_token_balance:#?}",);
```

Comment:
Remove

Reply:
but this log is in the `.mdx`?

Reply:
made the output look like the example given in `burn.mdx`

Reply:
Ah my bad then, though you were just adding it